build.assets/tooling/cmd/get-webapps-version/main.go

```diff
@@ -0,0 +1,34 @@
+// Command get-webapps-version determines the appropriate version
+// of webapps to check out for a given version of teleport.
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+)
+
+func main() {
+	fmt.Println(webappsVersion(
+		os.Getenv("DRONE_TAG"),
+		os.Getenv("DRONE_TARGET_BRANCH"),
+	))
+}
+
+func webappsVersion(tag, targetBranch string) string {
+	// if this build was triggered from a tag on the
+	// gravitational/teleport repo, assume that same
+	// tag exists on gravitational/webapps
+	if tag != "" {
+		return tag
+	}
+
+	// if this build is on one of the teleport release branches,
+	// map to the equivalent release branch in webapps
+	if strings.HasPrefix(targetBranch, "branch/") {
+		return "teleport-" + strings.TrimPrefix(targetBranch, "branch/")
+	}
```

Comment:
I don't love that this leads to a floating dependency (latest commit on the branch), which will then result in nondeterminism during any untagged builds.  That said, I understand this is an interim solution, and updating the equivalent of a lockfile is tedious.

Reply:
Yeah, this isn't great, but it at least ensures that connect continues to build successfully, so it's better than nothing which is the state we're in today.